### DISCUSSION TITLE
Configurable limit of sub claims that players can own inside a larger claim. (GPRealEstate)

### DISCUSSION THREAD

**Discussion by leviandstuff** [GriefPrevention/GriefPrevention#1080] [Ideas]
(Sorry if GPRealEstate is not supported here, I'm not sure where to get support for specific addons.)

As a survival server owner,
I would like to be able to put a limit on how many sub claims a player can own in a main claim with the GPRealEstate addon.
Because I would like to make towns where players can purchase up to 2 claims but are limited beyond that so that they cannot purchase every claim in the city. (I do not want to set incredibly high prices on the claims because these will be towns for new players.)

Optional questions:

**Describe alternatives you've considered**
I have considered giving players limited claim blocks so that they cannot own a ton of claims inside of a town. However, I would like players to have enough claim blocks to build their own builds outside of these towns.

**Comment by Jikoo** [comment]
While Robo does own the Techfortress org, you should be seeking GPRealEstate support at the GPRealEstate repo, not here.

That said, Robo does state in the readme that feature requests are available only to patrons, so maybe this should stay open here.

**Comment by leviandstuff** [comment]
> While Robo does own the Techfortress org, you should be seeking GPRealEstate support at the GPRealEstate repo, not here.
> 
> That said, Robo does state in the readme that feature requests are available only to patrons, so maybe this should stay open here.

Where can I find the GPRealEstate repo? I've found a bug with the plugin that I'd like to report.

**Comment by Jikoo** [comment]
I [already linked it to you](https://github.com/TechFortress/GriefPrevention/issues/1078#issuecomment-718146972) in a previous issue of yours, but [here it is again](https://github.com/RoboMWM/GPRealEstate).

**Comment by leviandstuff** [comment]
> I [already linked it to you](https://github.com/TechFortress/GriefPrevention/issues/1078#issuecomment-718146972) in a previous issue of yours, but [here it is again](https://github.com/RoboMWM/GPRealEstate).

Oh alright thanks! Wasn't sure what repo meant.

**Comment by Jikoo** [comment]
Oh, sorry. Repo is short for code repository.
